index.html

```diff
@@ -0,0 +1,208 @@
+                    <div class="card-body">
+                        <i class="bi bi-easel"></i>
+                        <h3 class="card-title">Exhibition</h3>
+                        <p class="lead">Online demonstrations will be taking place with our latest software dev tools.</p>
+                    </div>               
+                </div>
+                <div class="col-md-6 col-lg mt-3">                   
+                    <div class="card-body">
+                        <i class="bi bi-chat-right-text"></i>
+                        <h3 class="card-title">Forum</h3>
+                        <p class="lead">Any questions that you may have can be answered through numerous forums</p>
+                    </div>                   
+                </div>
+                <div class="col-md-6 col-lg mt-3">                   
+                    <div class="card-body">
+                        <i class="bi bi-gear"></i>
+                        <h3 class="card-title">Workshop</h3>
+                        <p class="lead">Sit down and work through our cutting edge online tech demos</p>
+                    </div>                   
+                </div>
+                <div class="col-md-6 col-lg mt-3">
+                    <div class="card-body">
+                        <i class="bi bi-magic"></i>
+                        <h3 class="card-title">TC Ignite</h3>
+                        <p class="lead">Network in an online setting that will further assist you in your business and career path</p>
+                    </div>
+                </div>
+            </div>
+
+            <div class="row">
+                <div class="col-md-12">
+                    <div class="section-header text-center p-5">
+                        <button type="button" class="btn btn-danger">See The Whole Program</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- Featured Speakers section -->
+
+    <section id="speaker-wrapper">
+        <h2 class="section-header text-center mt-4">Featured Speakers</h2>
+        <div class="guide_bar"></div>
+        <div class="container">
+            <div class="row p-2" id="featured-speakers">
+            </div>
+        </div>
+    </section>
+
+    <!-- Global sponsors -->
+
+    <section>
+        <h2 class="text-center text-dark mt-4">Partners</h2>
+        <div class="guide_bar"></div>
+        <ul class="sponsor-class">
+            <li><img class="col-md" id="sponsor-logo" src="./images/naver-logo.jpeg" alt="Naver logo"></li>
+            <li><img class="col-md" id="sponsor-logo" src="./images/daumkakao-logo.jpeg" alt="Daumkakao logo"></li>
+            <li><img class="col-md" id="sponsor-logo" src="./images/airbnb-logo.jpeg" alt="Airbnb logo"></li>
+            <li><img class="col-md" id="sponsor-logo" src="./images/mozilla-logo.jpeg" alt="Mozilla logo"></li>
+        </ul>
+
+    </section>
```

Comment:
- [OPTIONAL] According to the requirements and the original design, The Partners Section of the home page should has a dark background-color 

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/198776659-8c5dff3d-74f3-4a5f-a244-56065cfaa905.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/198777201-ee763d9b-0f8e-42a8-9b81-4b7c2a2bc427.png" width="500" /> |